Gemfile

```diff
@@ -5,6 +5,7 @@ source 'https://rubygems.org'
 gem 'rake', require: false
 # SDK feature dependencies
 gem 'aws-crt' if ENV['CRT']
+gem 'base64'
```

Comment:
Unrelated to this change, but should we add BigDecimal as well here?

Reply:
I'll do a separate PR to add ruby 3.4 to CI and also that.